apps/web/src/components/FiatOnRampModal/FiatOnRampModal.tsx

```diff
@@ -140,22 +152,9 @@ export const FiatOnRampModalButton = ({
       amount={amount}
     />,
   )
-
-  const { fiatOnarampAvailability, availabilityChecked, loading, error } = useFiatOnrampAvailability(
-    shouldCheck,
-    onPresentConfirmModal,
-  )
-
   const handleBuyCryptoClick = useCallback(() => {
-    if (!availabilityChecked) {
-      setShouldCheck(true)
-    } else if (fiatOnarampAvailability) {
-      onPresentConfirmModal()
-      setShouldCheck(false)
-    }
-  }, [fiatOnarampAvailability, availabilityChecked, onPresentConfirmModal])
-
-  const disableBuyCryptoButton = Boolean(error || (!fiatOnarampAvailability && availabilityChecked) || loading)
+    onPresentConfirmModal()
```

Comment:
Let's just use the `onPresentConfirmModal` without creating a new function variable if there's no extra logic here

Reply:
resolved